views/form/templates/preview.hbs

```diff
@@ -1,8 +1,11 @@
-<style type="text/css" id="mailpoet_form_preview_styles">{{{ css }}}</style>
+<style type="text/css" id="mailpoet_form_preview_styles">
+	.mailpoet_hp_email_label { display: none; }
+	{{{ css }}}
+</style>
 <div class="mailpoet_form">
   <div class="mailpoet_message">
-    <p class="mailpoet_validate_success"><%= __('This is a success message.') %></p>
-    <p class="mailpoet_validate_error"><%= __('This is an error message.') %></p>
+    <p class="mailpoet_validate_success"><%= __('This is how the success message will look like.') %></p>
+    <p class="mailpoet_validate_error"><%= __('This is how the error message will look like.') %></p>
```

Comment:
Change message to 'This is what the error message looks like.'

Reply:
Fixed.